### ISSUE TITLE
what's your experiment hyper-params?

### ISSUE THREAD

**Issue by guvcolie** [facebookresearch/dlrm#81]
Thank you for your excellent code!
Now, I am retraining your dlrm with criteo dataset. What hyper-params should I use to get the loss and accuracy results in the ReadMe?  By default in your code ?
batch size=
learning rate=
lr scheduler=
optimizer=
loss type=
Thank you!

**Comment by mnaumovfb** [comment]
We use SGD optimizer in this code. You can find the remaining parameters in the `./bench/dlrm_s_criteo_kaggle.sh` shell script.

**Comment by guvcolie** [comment]
OK.  I see you set batch size=128, the number of the whole training set is 39291958, so I will need 39291958/128=306968 iters to iterate over the training set once, while you just put 1e5 iters results of Criteo on ReadMe. right?

**Comment by mnaumovfb** [comment]
Notice that
```
python
>>> print("{:.1e}".format(39291958/(128.)))
3.1e+05
```

**Comment by guvcolie** [comment]
My fault...
